packages/nextjs/tailwind.config.js

```diff
@@ -71,6 +71,9 @@ module.exports = {
   },
   theme: {
     extend: {
+      colors: {
+        customBlue: '#D5DAFF',
```

Comment:
I think before tackling #32, maybe we could review our color pallet once and make it inline with figma (Thanks to Andrea she nicely named all colors)

And it will also make #32 easy 🙌